src/components/Image/ImageLoader.jsx

```diff
@@ -30,6 +38,7 @@ class ImageLoader extends React.Component {
     if (status === PENDING) this.loadLink()
     else if (status === LOADING_LINK) this.loadFallback()
     else if (status === LOADING_FALLBACK) {
+      console.warn('failed loading thumbnail', lastError)
```

Comment:
I suspect we'll get some cases of "why don't I have thumbnails?", this will allow us to find answers in the logs. We can remove this once the feature is stable.